### PR TITLE
Make URLs used in UI fully configurable

### PR DESCRIPTION
**Pull request by jasonhansel**

### CODE REVIEW

File: properties/config.json [hunk 1]
```diff
@@ -118,7 +126,11 @@
       },
       "EPIC": false,
       "cBio": false,
+      "linkToHomePage": "https://matchminer.dfci.harvard.edu",
```
Comment (jim-bo) [inline properties/config.json:129]: Do we want these links in the public site? I see the request access link to google form was already there? Do we want to strip that?

**Review by jim-bo — COMMENTED** [review]
Hey, just wanted to check on those links...

File: properties/config.json (hunk 1 above)
Comment (jasonhansel) [reply properties/config.json:129]: These links were already there, and they don't apply to the setup script, which installs its own configuration in place of this one. So this configuration is just to keep the code working outside of the setup script, within DFCI.

**Review by Luke-Sikina — APPROVED** [review]

**Review by galvinma — APPROVED** [review]

**Comment by galvinma** [comment]
LGTM

**Review by jim-bo — APPROVED** [review]
thanks I appreciate it

**Pull request merged**